### PR TITLE
Update 2022-04-18-eia-roadmap-update.md

### PR DESCRIPTION
**Pull request by brynnelliott**
tl;dr - is jargon, I recommend using plain language

Please delete this line and the text below before submitting your contribution.

---

Thanks for contributing! If this contribution is for instructor training, please send an email to checkout@carpentries.org with a link to this contribution so we can record your progress. You’ve completed your contribution step for instructor checkout just by submitting this contribution.

Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.  If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact The Carpentries Team at team@carpentries.org.

---

### CODE REVIEW

**Comment by ErinBecker** [comment]
Thanks @brynnelliott! I agree with this recommendation and would love to see this reflected in [our style guide](https://docs.carpentries.org/topic_folders/communications/resources/style-guide.html#).

**Comment by maneesha** [comment]
Thanks @ErinBecker  and @brynnelliott  I've opened an issue in the Handbook repo.
